### PR TITLE
Put deps into version catalog

### PR DESCRIPTION
**Pull request by Goooler**
After this, we can convert kts easily.

### CODE REVIEW

**Comment by JakeWharton** [comment]
Please do not convert this project to kts. We do not want it.

File: settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ":retrofit-converters:moshi",
+  ":retrofit-converters:protobuf",
+  ":retrofit-converters:scalars",
+  ":retrofit-converters:simplexml",
+  ":retrofit-converters:wire",
+
+  ":samples",
+)
+
+enableFeaturePreview("VERSION_CATALOGS")
```
Comment (JakeWharton) [inline settings.gradle.kts:31]: Revert change to kts and add this to old file.

**Review by JakeWharton — APPROVED** [review]

File: settings.gradle.kts (hunk 1 above)
Comment (Goooler) [reply settings.gradle.kts]: Done.

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**